src/physics/collision_object.c

```diff
@@ -114,16 +114,10 @@ void collisionObjectHandleSweptCollision(struct CollisionObject* object, struct
     if (velocityDot < 0.0f) {
         vector3AddScaled(&object->body->velocity, normal, (1 + restitution) * -velocityDot, &object->body->velocity);
         vector3AddScaled(&object->body->transform.position, normal, -0.01f, &object->body->transform.position);
+    }
 
-        // I don't love that player specific code
-        // is in the general collision code but 
-        // that is how it is for now
-        // a better general solution may be to allow any
-        // object to determine if it had any impacts
-        // and respond dynamically
-        if (object->body->flags & RigidBodyIsPlayer) {
-            playerHandleLandingRumble(-velocityDot);
-        }
+    if (object->sweptCollide) {
```

Comment:
Minor: can you move this back into the if body? We should only get here if the objects are moving toward each other (i.e., not already handled/separated).